docs/tutorials/index.rst

```diff
@@ -0,0 +1,11 @@
+Tutorials
+=======================================
+
+.. toctree::
+   :maxdepth: 1
+
+   dev-setup
+   blockchain     
+   first-transaction
+   proxy-tutorial
```

Comment:
Consider using "Paywall tutorial" as a name. "Proxy tutorial" doesn't say anything to the user.

Reply:
Fixed in: https://github.com/raiden-network/microraiden/commit/1d95ca2ee50c62bfc0efc17791dcd8e5eafb424a